packages/core/src/auto.ts

```diff
@@ -434,11 +437,15 @@ export default class Auto {
       }
     } else if (editFlag && message) {
       await this.git.editComment(message, prNumber, context);
-      this.logger.log.success(`Edited comment on PR #${prNumber} under context "${context}"`);
+      this.logger.log.success(
```

Comment:
Any reason for all of the formatting changes?

Reply:
yarn lint is doing that. Shall i change it back?

Reply:
I think it's cause in another PR I used GitHub to edit these lines, so they didn't get formatted

Reply:
Oh i thought it is caused by linting.